docs/user/config/isolatedModules.md

```diff
@@ -43,3 +43,31 @@ module.exports = {
 ```
 
 </div></div>
+
+# Performance
+
+Using `isolatedModules: false` comes with a cost of performance comparing to `isolatedModules: true`. There is a way
+to improve the performance when using this mode by changing the value of `include` in `tsconfig` which is used by `ts-jest`.
+The least amount of files which are provided in `include`, the more performance the test run can gain.
+
+### Example
+
+```json5
+// tsconfig.json
+{
+  // ...other configs
+  include: [
+    "my-typings/*",
+    "my-global-modules/*",
+  ]
+}
+```
+
+## Caveats
```

Comment:
not sure if writing like this is fine. Would you please check ?

Reply:
This looks fine. It's basically your opinion on how this could be helpful under specific conditions. Nothing wrong with it